Add description (#519)
harness/developer-hub: docs/chaos-engineering/chaos-faults/kubernetes/pod/pod-cpu-hog-exec.md

<summary>View fault usage</summary>
<div>

> Disk pressure or CPU hog affects Kubernetes applications that results in the eviction of the application replica and impacts its delivery. These issues are referred to as "noisy neighbour" problems.

Reviewer: ```suggestion
CPU hog affects Kubernetes applications which result in the eviction of the application replica and impacts its delivery. These issues are referred to as "noisy neighbour" problems.
```
